molecularnodes/nodes/material.py

```diff
@@ -187,3 +189,249 @@ def __init__(self, **kwargs):
 class TransparentOutline(MaterialConstructor):
     def __init__(self, **kwargs):
         super().__init__("MN Transparent Outline", **kwargs)
+
+
+class MN_Materials:
+    """
+    Note: if this is a nice route we could add fns that will create MN_materials here and remove them from the blend file.
+
+    Materials:
+    - AmbientOcclusion
+    - Default  ( tried to replicate the defaults below )
+    - FlatOutlineq
+    - Squishy
+    - TransparentOutline
+
+    """
+
+    @staticmethod
+    def glass() -> bpy.types.Material:
+        return create_material(
+            name="MN_Glass",  # Add a descriptive name
```

Comment:
For the individual material names, overall I have been thinking of removing the `MN ` prefix from the other materials, which I have done for all of the user-facing nodes inside of Geometry Nodes.

For now I think we stick with the prefix, but I think it should use space instead of underscore. Could also be something that is prepended by the `create_material()` function rather than having to define it in the actual name argument.

Reply:
Ok. Converted to space.s